source/constructs/api/catalog/sample_service.py

```diff
@@ -35,11 +38,10 @@ def get_sample_file_uri(database_type: str, database_name: str, table_name: str)
     result_table = const.JOB_SAMPLE_RESULT_TABLE_NAME
     full_database_name = f"{database_type}-{database_name}-database"
     file_folder_path = f"glue-database/{result_table}/{full_database_name}/{table_name}/"
-    bucket_name = os.getenv(const.PROJECT_BUCKET_NAME, const.PROJECT_BUCKET_DEFAULT_NAME)
     file_uri = ''
     creation_time = ''
     s3 = boto3.client('s3')
-    response = s3.list_objects_v2(Bucket=bucket_name,
+    response = s3.list_objects_v2(Bucket=admin_bucket_name,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `list_objects_v2` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `ContinuationToken`, `IsTruncated`, `NextContinuationToken`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)